packages/api/bin/cli.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/* eslint-disable no-console */
+
+const pckg = require('../package.json');
+const es = require('./es');
+const program = require('commander');
+const { cliUtils } = require('@cumulus/common');
+
+program.version(pckg.version);
+
+program
+  .usage('TYPE COMMAND [options]');
+
+program
+  .command('reindex')
+  .description('Reindex elasticsearch index to a new destination index')
+  .option('-a, --index-alias <alias>', 'AWS Elasticsearch index alias', 'cumulus-alias')
+  .option('--host <host>', 'AWS Elasticsearch host', null)
+  .option('-s, --source <sourceIndex>', 'Index to reindex', null)
+  .option('-d, --dest-index <destIndex>',
+    // eslint-disable-next-line max-len
```

Comment:
nit: I tend to put `// eslint-disable-line max-len` at the end of the offending line instead of having a comment above it.  I tend to think that it makes the comment less obtrusive.